packages/cardano-services/src/ChainHistory/DbSyncChainHistory/mappers.ts

```diff
@@ -105,14 +105,30 @@ export const mapWithdrawal = (withdrawalModel: WithdrawalModel): Cardano.Withdra
 // Remove this and select the actual redeemer data from `redeemer_data` table.
 const stubRedeemerData = Buffer.from('not implemented');
 
+const mapRedeemerPurpose = (purpose: string): Cardano.RedeemerPurpose => {
+  switch (purpose) {
+    case 'cert':
+      return Cardano.RedeemerPurpose.certificate;
+    case 'mint':
+      return Cardano.RedeemerPurpose.mint;
+    case 'spend':
+      return Cardano.RedeemerPurpose.spend;
+    case 'reward':
+      return Cardano.RedeemerPurpose.withdrawal;
+    default:
+      // TODO: find what value it can be
+      return Cardano.RedeemerPurpose.delegateRepresentative;
```

Comment:
@mirceahasegan / @iccicci what is the correct epic / ticket to add this for Conway era support work?

Reply:
LW-9199 is the correct epic.

Reply:
FYI: this was already done, in a similar form on conway-era branch, so the master->conway-era merge will have to be manually resolved.